source/isaaclab/isaaclab/assets/rigid_object/rigid_object.py

```diff
@@ -292,6 +294,8 @@ def write_root_velocity_to_sim(self, root_velocity: torch.Tensor, env_ids: Seque
             root_velocity: Root center of mass velocities in simulation world frame. Shape is (len(env_ids), 6).
             env_ids: Environment indices. If None, then all indices are used.
         """
+        if self.is_kinamtic_enabled:
```

Comment:
Calling the variable just "is_kinematic" is sufficiently concise. Don't need to say enabled.

Reply:
Also this can be a "data" attribute? You should be able to get it from the PhysX view for individual prims.